Header header

uint8 status
uint8 error

uint8 DONE=0
uint8 DOING=1
uint8 FINISHING=2

uint8 GOING_WELL=0
uint8 IN_ROCK=1
uint8 PATH_NOT_FOUND=2
